nanotune/device/device.py

```diff
@@ -61,16 +190,43 @@ def __init__(
         channels: Optional[
             Union[Mapping[str, Mapping[str, Any]], Mapping[str, str]]] = None,
         readout: Optional[Mapping[str, str]] = None,
+        main_readout_method: Union[ReadoutMethods, str] = ReadoutMethods.transport,
         initial_values: Optional[Mapping[str, Any]] = None,
         set_initial_values_on_load: bool = False,
         device_type: Optional[str] = '',
         initial_valid_ranges: Optional[Mapping[str, Sequence[str]]] = None,
         current_valid_ranges: Optional[Mapping[str, Sequence[str]]] = None,
-        normalization_constants: Optional[nrm_cnst_tp] = None,
+        normalization_constants: Optional[
+            Union[Dict[str, Sequence[float]], NormalizationConstants]] = None,
         transition_voltages: Optional[Mapping[str, float]] = None,
         **kwargs,
     ) -> None:
-
+        """Device init method.
+
+        Args:
+            name (str): instrument/device name.
```

Comment:
This repeats documentation from above. I would omit one of them otherwise they are very likely to go out of sync

Reply:
As discussed on teams, both docstrings make sense as the init takes different arguments than what the attributes are later (due to parameter and channel instantiation similar to the DelegateInstrument).